src/components/portfolio/DetailsMainBacktestingList.tsx

```diff
@@ -74,25 +74,27 @@ const DetailsMainBacktestingList: React.FC<DetailsMainBacktestingListProps> = ({
       <DetailsMainBacktestingListItem
         icon={<HeightIcon className={classes.mdd} />}
         title={t('portfolio.details.backtesting.mddMinMax')}
-        value={<StyledNumberFormat value={mddMaxToMin} suffix="%" />}
+        value={<StyledNumberFormat value={mddMaxToMin * 100} suffix="%" />}
         infoText={t('portfolio.details.backtesting.mddInfo')}
       />
       <DetailsMainBacktestingListItem
         icon={<WarningIcon className={classes.beta} />}
         title={t('portfolio.details.backtesting.standardDeviation')}
-        value={<StyledNumberFormat value={standardDeviation} suffix="%" />}
+        value={
+          <StyledNumberFormat value={standardDeviation * 100} suffix="%" />
+        }
         infoText={t('portfolio.details.analytics.standardDeviation.infoButton')}
       />
       <DetailsMainBacktestingListItem
         icon={<NewReleasesIcon className={classes.sharpe} />}
         title={t('portfolio.details.backtesting.sharpeRatio')}
-        value={sharpeRatio}
+        value={<StyledNumberFormat value={sharpeRatio} suffix="" />}
```

Comment:
`suffix=""` was not introduced in this PR, but it is actually misleading because the suffix is a space rather than an empty string. This should be fixed in `StyledNumberFormat`.

I would make `suffix` optional ('€' or '%'). If not set, no suffix should be added (also no space, I don't think this was intentional).